api/backend.go

```diff
@@ -373,7 +373,7 @@ func (b *StatusBackend) Logout() error {
 	switch err {
 	case node.ErrServiceUnknown: // Whisper was never registered
 	case nil:
-		if err := whisperService.DeleteKeyPairs(); err != nil {
+		if err = whisperService.DeleteKeyPairs(); err != nil {
```

Comment:
I guess the previous version was fine. Do we want to avoid variable shadowing overall?

Reply:
I prefer the previous version allowing variable shadowing.

Reply:
OK, I'll revert those changes then.